iothub/service/src/RetryPolicies/IotHubServiceRetryPolicyBase.cs

```diff
@@ -57,7 +57,7 @@ protected TimeSpan UpdateWithJitter(double baseTimeMs)
             lock (_rngLock)
             {
                 // A random double from 95% to 105% of the baseTimeMs
-                jitterMs = _rng.Next(95, 106);
+                jitterMs = _rng.Next(95, 105);
```

Comment:
yes, since the unit test fails from values at 6% over the average are outside of the boundary. The alternative could be changing the test itself to accept up to 1.06x the user set value.

Reply:
Change the test case.